src/frontend/pages/_document.tsx

```diff
@@ -0,0 +1,57 @@
+import { ServerStyleSheets } from "@material-ui/core/styles";
+import Document, {
+  DocumentContext,
+  DocumentInitialProps,
+  Head,
+  Html,
+  Main,
+  NextScript,
+} from "next/document";
+import React from "react";
+import { theme } from "../src/common/styles/theme";
+
+export default class MyDocument extends Document {
+  static async getInitialProps(
+    context: DocumentContext
+  ): Promise<DocumentInitialProps> {
+    const sheets = new ServerStyleSheets();
```

Comment:
This is needed to set up MUI's server side rendering capability: https://material-ui.com/guides/server-rendering/#server-rendering